### PR TITLE
[Feature Request] Close terminal tab with Scroll Wheel #88

### PR DESCRIPTION
**Pull request by sylank**
Close terminal tab with Scroll Wheel

### CODE REVIEW

File: lib/config.vala [hunk 1]
```diff
@@ -108,7 +108,7 @@ namespace Config {
                 config_file.set_string("shortcut", "close_window", "Ctrl + Alt + q");
                 config_file.set_string("shortcut", "close_other_windows", "Ctrl + Shift + q");
 
-                config_file.set_string("shortcut", "rename_title", "F2");
+                config_file.set_string("shortcut", "rename_title", "Alt + t");
```
Comment (BLumia) [inline lib/config.vala]: Please keep a commit atomic, don't make any change not related to the current Pull Request

File: lib/config.vala [hunk 2]
```diff
@@ -70,10 +70,10 @@ namespace Config {
             }
 
             var file = File.new_for_path(config_file_path);
-            if (!file.query_exists()) {
-                init_config();
-            } else {
+            if (file.query_exists()) {
                 load_config();
+            } else {
+                init_config();
```
Comment (BLumia) [inline lib/config.vala]: Seems this is not a necessary change, keep the old code as-is unless it's really need to be modify :)

**Review by BLumia — CHANGES_REQUESTED** [review]
A couple of inline comments.

File: lib/config.vala (hunk 1 above)
Comment (sylank) [reply lib/config.vala]: sorry, accidentally I copied this changes from my another branch :)

File: lib/config.vala (hunk 2 above)
Comment (sylank) [reply lib/config.vala]: I just wanted to clean that if statement, but yes this modification is not part of this scope. reverted :)

**Review by BLumia — APPROVED** [review]
Looks good.

**Pull request merged**